app/src/main/res/layout/character_detail_layout.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
where did you use this xml?

Reply:
That xml was created to be used in the next commit, in which the functionality of the fd will be done.

Reply:
Okay, If this file is not in the scope of the PR I think you need to upload it in the next PR